server/protocols/jmap-draft/src/main/java/org/apache/james/jmap/http/AuthenticationRoutes.java

```diff
@@ -102,10 +102,25 @@ public Logger logger() {
     @Override
     public Stream<JMAPRoute> routes() {
         return Stream.of(
-            new JMAPRoute(new Endpoint(HttpMethod.POST, AUTHENTICATION), JMAPRoutes.corsHeaders(this::post)),
-            new JMAPRoute(new Endpoint(HttpMethod.GET, AUTHENTICATION), JMAPRoutes.corsHeaders(this::returnEndPointsResponse)),
-            new JMAPRoute(new Endpoint(HttpMethod.DELETE, AUTHENTICATION), JMAPRoutes.corsHeaders(this::delete)),
-            new JMAPRoute(new Endpoint(HttpMethod.OPTIONS, AUTHENTICATION), CORS_CONTROL)
+            JMAPRoute.builder()
```

Comment:
As all parameters are required, you can bypass `.build` and use a stage builder

Reply:
The cors one is not required?

Reply:
Really? It looks like there's two choices for cors but no case of "do nothing", right?